e2e/apprunner/apprunner_test.go

```diff
@@ -180,12 +205,17 @@ var _ = Describe("App Runner", func() {
 			}
 		})
 
-		It("should return svc endpoints", func() {
+		It("should return svc endpoints and service discovery should work", func() {
```

Comment:
niceee

Can we also add `copilot storage init -t Aurora` to the "front-end" service. I think both SD and RDS are common usecases that we'll want to make sure works for RDWS? 🙏

Reply:
Maybe something like `http.Get(fmt.Sprintf("%s/db", route.URL))` which returns a timestamp from the database.

For example, in JavaScript I had this working:
```js
app.get('/db', async (req, res) => {
  const dbSecret = await client.getSecretValue({SecretId: process.env.WWWCLUSTER_SECRET_ARN}).promise();
  const {username, host, dbname, password, port} = JSON.parse(dbSecret.SecretString);
  try {
    const client = new Client({
      user: username,
      host: host,
      database: dbname,
      password: password,
      port: port,
    });
    client.connect();
    const response = await client.query('SELECT NOW() as now');
    
    res.send(`Hello results: ${JSON.stringify(response.rows[0])}`);
  } catch (err) {
    res.send(`db error: ${JSON.stringify(err)}`);
  };
});
```

Reply:
Seems like even after the VPC support for App runner, there's still no native way for us to support aurora with our RDWS because app runner doesn't support secrets natively 🤯 they would need to use aws sdk in their code which require extra addons permissions to get the secret instead of directly using it, which I won't expect it to be a very popular pattern. Could we add it once app runner supports secrets from ssm and secret manager?

Reply:
ok addressed